shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -57,11 +57,11 @@
 </template>
 
 <template name="_loginButtonsLoggedOutDropdown">
-  <h4>Sign in</h4>
-
   {{#if isDemoUser}}
-    <p class="demo-note">Sign in to make your demo data permanent.</p>
+    <h4>Create Account</h4>
+    <p class="demo-note">Your demo data will be transferred to your new account.</p>
   {{else}}
+    <h4>Sign in</h4>
```

Comment:
But >99% of humanity doesn't have an account yet. But I am OK with leaving this as "Sign in" for now.